bindings/python/examples/gesture_rec/notebook.py

```diff
@@ -0,0 +1,254 @@
+
+from tkinter import *
+from tkinter import ttk
+import aditofpython as tof
+import numpy as np
+import cv2 as cv
+from process import ProcessTab
+
+smallSignalThreshold = 100
+
+config_options = [
+        "config/config_adsd3500_adsd3100.json",
+        "config/config_adsd3500_adsd3030.json"
+]
+
+mode_options = [
+        "sr-qnative",
+        "lr-qnative",
+        "sr-native",
+        "lr-native"
+]
+
+# Global variables for the program
+config_options_chosen = config_options[0]
+mode_options_chosen = mode_options[0]
```

Comment:
can we parse the mode, config file and ip via CLI?

Reply:
You can find OptionList-s with Dropdown capability to select the desired one